livekit-plugins/install_local.sh

```diff
@@ -19,4 +19,5 @@ pip install \
   "${SCRIPT_DIR}/livekit-plugins-rag" \
   "${SCRIPT_DIR}/livekit-plugins-playai" \
   "${SCRIPT_DIR}/livekit-plugins-silero" \
-  "${SCRIPT_DIR}/livekit-plugins-turn-detector"
+  "${SCRIPT_DIR}/livekit-plugins-turn-detector" \
```

Comment:
Btw we already have install-plugins-editable.sh not sure how this script was committed

Reply:
we need two versions because editable is not the same thing as local install. mypy requires latter